src/Imdb/Calendar.php

```diff
@@ -0,0 +1,92 @@
+<?php
+
+#############################################################################
+# IMDBPHP                              (c) Giorgos Giagas & Itzchak Rehberg #
+# written by Giorgos Giagas                                                 #
+# extended & maintained by Itzchak Rehberg <izzysoft AT qumran DOT org>     #
+# http://www.izzysoft.de/                                                   #
+# ------------------------------------------------------------------------- #
+# IMDBPHP TOP CHARTS                                      (c) Ricardo Silva #
+# written by Ricardo Silva (banzap) <banzap@gmail.com>                      #
+# http://www.ricardosilva.pt.tl/                                            #
+# ------------------------------------------------------------------------- #
+# IMDBPHP UPCOMMING RELEASES (based on IMDBPHP TOP CHARTS (c)Ricardo Silva) #
+# written/modified/extended by Ed (github user: duck7000)                   #
+# ------------------------------------------------------------------------- #
+# This program is free software; you can redistribute and/or modify it      #
+# under the terms of the GNU General Public License (see doc/LICENSE)       #
+#############################################################################
+
+namespace Imdb;
+
+/**
+ * Obtaining information about upcomming movie releases as seen on imdB
+ * https://www.imdb.com/calendar
+ * @author Ricardo Silva (banzap) <banzap@gmail.com>
+ * @author Ed (github user: duck7000)
+ */
+class Calendar extends MdbBase
+{
+
+    protected $page = null;
```

Comment:
This isn't used
```suggestion
```

Reply:
This was in Charts.php which i used as base so assumed it was needed. Removed now.